pkgs/development/libraries/git2/0.27.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, python
+, zlib, libssh2, openssl, http-parser, curl
+, libiconv, Security
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
can you just do a `libgit2_0_27 = libgit.overrideAtts…` inside a `let` expression inside `gitaly/default.nix`, that simply overwrites `src`? That way it's clear where this is used, and we avoid duplicating this whole file.